packages/xo-web/src/xo-app/backup/overview/index.js

```diff
@@ -1,273 +1,376 @@
 import _ from 'intl'
-import ActionRowButton from 'action-row-button'
-import ButtonGroup from 'button-group'
-import Component from 'base-component'
+import ActionButton from 'action-button'
```

Comment:
Has anything else than the legacy overview changed in this file?

Reply:
No.